src/test/java/com/twilio/twiml/FaxResponseTest.java

```diff
@@ -80,4 +80,53 @@ public void testElementWithTextNode() {
             elem.toXml()
         );
     }
+
+
+    @Test
+    public void testElementWithGenericNode() {
+        FaxResponse.Builder builder = new FaxResponse.Builder();
+
+        GenericNode.Builder genericBuilder = new GenericNode.Builder("genericTag");
+
+        GenericNode node = genericBuilder.build();
+
+        FaxResponse elem = builder.addChild(node).build();
+
+        Assert.assertEquals(
+                "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
+                        "<Response>" +
+                        "<genericTag/>"+
+                        "</Response>",
+                elem.toXml()
+        );
+    }
+
+    @Test
+    public void testElementWithGenericNodeAttributes() {
+        FaxResponse.Builder builder = new FaxResponse.Builder();
+
+        GenericNode.Builder genericBuilder = new GenericNode.Builder("genericTag");
+
+        genericBuilder.option("key", "value").addText("someText");
+
+        GenericNode node = genericBuilder.build();
```

Comment:
I'd find it more readable if we combine some of these lines and put them in little sections as we're constructing each object, from bottom up:
```
GenericNode.Builder genericBuilder = new GenericNode.Builder("genericTag");
GenericNode node = genericBuilder.option("key", "value").addText("someText").build();

Receive.Builder receiveBuilder = new Receive.Builder();
Receive message = receiveBuilder.addChild(node).build();

FaxResponse.Builder builder = new FaxResponse.Builder();
FaxResponse elem = builder.receive(message).build();
```